### DISCUSSION TITLE
Custom build of extension

### DISCUSSION THREAD

**Discussion by mobot95** [crazywhalecc/static-php-cli#510] [Q&A] [answered]
I didn't find any reference regarding manually compiling of extensions.
I'd like to include CURL with ARES (adding --enable-ares), any chance of doing it using spc?

**Comment by crazywhalecc** [comment, marked as answer]
I haven't added ares support for libcurl, but it's easy to add it. And BTW the config files (`config/xxx.json`) are important parts to read if you want to fully customize static-php-cli.

I'll draft a PR to update this tomorrow, and then you can use `build curl --with-suggested-libs` to use curl with libcares.
